style.css

```diff
@@ -96,3 +96,272 @@ a {
   background-position-x: right;
   background-position-y: 120px;
 }
+
+.recent-work {
+  width: 375px;
+  height: 6080px;
+}
```

Comment:
- [ ] Please make use of a CSS grid to layout the projects in this section. CSS grid helps you easily layout your cards based on screen sizes you tell it to. For instance, you can make things display a grid and then specify the number of columns to show based on the screen size you are working on. Check this guide on [How to use CSS Grid](https://www.w3schools.com/css/css_grid.asp) 👍

Reply:
resolved 👍